databricks_cli/dbfs/api.py

```diff
@@ -142,11 +180,16 @@ def get_num_files_deleted(partial_delete_error):
                     message))
         return int(m.group(1))
 
+    @retry_429
+    def delete_with_retries(self, dbfs_path, recursive, headers):
```

Comment:
I think you forgot to remove the `_with_retries` from the method name, since we're not using this anywhere else. Or did you keep it for delete intentionally?